pages/common/gource.md

```diff
@@ -0,0 +1,31 @@
+# gource
+
+> Visualize the history of Git, SVN, Mercurial and Bazaar repositories.
+
+- Run gource in a directory:
+
+`gource {{/path/to/repository/}}`
+
+- Run gource in fullscreen mode:
```

Comment:
I'd mention that this assumes the command is run from the project's root directory.